odoo/addons/test_new_api/tests/test_many2many.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import MissingError
+from odoo import Command
+
+
+class many2manyCase(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        pirate = self.env["test_new_api.pirate"].create({'name': 'Barbe noire'})
+        self.env["test_new_api.pirate"].create({'name': 'Barbe rouge'})
+        prisonner = self.env['test_new_api.prisonner'].create({'name': 'Brian'})
+        self.ship = self.env["test_new_api.ship"].create({'name': 'Colombus'})
+        pirate.ship_ids = self.ship
+        prisonner.ship_ids = self.ship
+
+    def test_new_many_2_many(self):
```

Comment:
This name is kinda useless, `test_new` means nothing, and `many_2_many` is already the name of the current test class

Ideally name should contain the not in/multi relation thing on it.